http-get-product/index.ts

```diff
@@ -0,0 +1,17 @@
+import { AzureFunction, Context, HttpRequest } from "@azure/functions"
+
+const httpTrigger: AzureFunction = async function (context: Context, req: HttpRequest): Promise<void> {
```

Comment:
looks like this is just an example function. I guess you can delete it

Reply:
Removed example